packages/schema/docs/build/schema.md

```diff
@@ -593,6 +596,34 @@ Key | Required | Type | Description
 
 -----
 
+## /BulkObjectSchema
+
+Zapier uses this configuration for writing in bulk.
+
+#### Details
+
+* **Type** - `object`
+* [**Source Code**](https://github.com/zapier/zapier-platform/blob/zapier-platform-schema@15.7.2/packages/schema/lib/schemas/BulkObjectSchema.js)
+
+#### Properties
+
+Key | Required | Type | Description
+--- | -------- | ---- | -----------
+`groupedBy` | **yes** | `array` | The list of keys of input fields to group bulk-write with. The actual user data provided for the fields will be used during execution. Note that a required input field should be referenced to get user data always.
+`limit` | **yes** | `integer` | The maximum number of items to call performBulk with.
```

Comment:
May be helpful (but also too lengthy here) to explain:

1. How to decide on a good value, based on max payload size.
2. That there's also a time/window limit, which we manage.